README.md

```diff
@@ -1,12 +1,53 @@
-## How to set navigation
-- Create a directory hierarchy in `src/pages/config.md`
+# Adobe Connect SDK Documentation
 
-## Launching a deploy
-- Go to Actions > Deployment > Run workflow
+Adobe Connect SDK documentation site deployed to EDS.
+The production address is https://developer.adobe.com/adobe-connect-sdk/
 
-## URL
-- developer-stage.adobe.com/{path-prefix}/{relative-path-to-file}
+## Quick Start
 
-## Where to ask for help
+Start development server:
+```bash
+npm run dev
+```
+Runs Express server on port 3003 serving files from `src/pages/`.
```

Comment:
Users should only navigate to `localhost:3000` for the intended local dev experience